MvvmCross/iOS/iOS/Views/Presenters/Attributes/IMvxOverridePresentationAttribute.cs

```diff
@@ -0,0 +1,8 @@
+﻿using System;
+namespace MvvmCross.iOS.Views.Presenters.Attributes
+{
+    public interface IMvxOverridePresentationAttribute
+    {
+        MvxBasePresentationAttribute OverridePresentationAttribute();
```

Comment:
I think the name `OverridePresentationAttribute` doesn't need the word override since you already see that in the interface.